tests/test_tasker.py

```diff
@@ -298,3 +298,12 @@ def test_build_image_from_git(temp_image_name):
     assert response is not None
     assert t.image_exists(temp_image_name)
     t.remove_image(temp_image_name)
+
+
+def test_get_info():
+    if MOCK:
+        mock_docker()
+
+    t = DockerTasker()
+    response = t.get_info()
+    assert response is not None
```

Comment:
How about `assert isinstance(response, tuple)`, `assert len(response) == 2`, and verifying each is a dict?